chia/types/eligible_coin_spends.py

```diff
@@ -326,20 +335,15 @@ async def process_fast_forward_spends(
             new_coin_spends.append(new_coin_spend)
         if len(ff_bundle_coin_spends) == 0:
             # This item doesn't have any fast forward coins, nothing to do here
-            return
+            return bundle_coin_spends
         # Update the mempool item after validating the new spend bundle
         new_sb = SpendBundle(
             coin_spends=new_coin_spends, aggregated_signature=mempool_item.spend_bundle.aggregated_signature
         )
         # We need to run the new spend bundle to make sure it remains valid
         assert mempool_item.conds is not None
         try:
-            new_conditions = get_conditions_from_spendbundle(
```

Comment:
why is it safe to ignore the new `SpendBundleConditions`? are they expected to be the same?
What's the point of calling `get_conditions_from_spendbundle()` if we just throw away the results? (the answer should be a comment in the code, I think)

Reply:
We just want the bundle to remain valid after processing fast forwards, not to break the mempool bundle later on at block creation time. If there is a better way to validate it nowadays, I'm happy to adopt it.